### DISCUSSION TITLE
How to set the gaps in special workspace to zero?

### DISCUSSION THREAD

**Discussion by Troublor** [hyprwm/Hyprland#3497] [Q&A] [answered]
I tried to configure the workspace rule for the special workspace, by setting `gapsin:0` and `gapsout:0`. But the gap is still huge. It seems the gaps configuration and rendering in special workspaces is different from those in normal workspaces.
Is it a bug?

**Comment by Troublor** [comment, marked as answer]
OK I found it. `special_scale_factor` in the Dwindle config will do the trick.

**Reply by ionicmc-js** [reply]
i think you set it to 1 to get 0 gaps
